stellar-core/debian/changelog

```diff
@@ -1,3 +1,9 @@
+stellar-core (_VERSION_-_BUILD_VERSION_) stable; urgency=medium
+
+  * added cargo rustc Build-Depends
+
+ -- Package Maintainer <packages@stellar.org>  Fri, 30 Apr 2022 01:05:00 +0000
+
```

Comment:
This will interfere slightly with our build process. We use `dch` command to add changelog entry on every build and it will result in invalid changelog file because of the `_VERSION_` placeholder.
We could add changelog entry with a fixed version to work around this? Maybe `19.0.0` to indicate the dependency change